sys/include/net/gnrc/gomach/gomach.h

```diff
@@ -0,0 +1,338 @@
+/*
+ * Copyright (C) 2016 INRIA
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    net_gnrc_gomach A traffic-adaptive multi-channel MAC
+ * @ingroup     net_gnrc
+ * @brief       A traffic adaptive MAC protocol that provides high traffic adaptability,
```

Comment:
Here and in other places: Please stick to the coding conventions and aim for a line-length of 80 characters (with 100 as an absolute maximum)